### ISSUE TITLE
Balloon grab alert incorrect at wall

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#296]
While reviewing description document, it looks like the description when grabbing the balloon at the wall is not correct. It is missing charge information about the wall.

**Comment by jessegreenberg** [comment]
Location  of charges when describing induced charge is now incorrect. I am seeing things like "Charges in center of play area..."

**Comment by jessegreenberg** [comment]
That part fixed in the above commit... Continuing review of grabbed alerts.

**Comment by jessegreenberg** [comment]
Above commit fixes the 'Grabbed' alerts in "Showing all charges" cases.  I am testing out a new way of assembling long descriptions, instead of having a bunch of string patterns, I have one string pattern and then modify it when I need to pull placeholders out for different cases. I think it makes the code easier to read.

**Comment by jessegreenberg** [comment]
Above commit makes  the grabbed alert match the design doc. Punctuation  is messed up, once that is fixed this can  be closed.

**Comment by jessegreenberg** [comment]
We are in line with design doc! Closing.
